main-site/components/adventure-ahead-section/AdventureAheadSection.styles.ts

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+import { P } from '../../../shared-ui/style/typography';
+import { max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledAdventureAheadSectionContainer = styled.div`
+  display: flex;
+  padding-top: 16em;
+  margin: 10em 0;
+  @media ${max.tablet} {
```

Comment:
Currently, this section looks too far to the left on desktop - adding margin on desktop will help make it more centered 
<img width="1512" alt="Screen Shot 2022-10-08 at 8 25 43 PM" src="https://user-images.githubusercontent.com/59738880/194732099-6c28c6e1-c416-4512-8f07-9162cafefdaa.png">


`const StyledAdventureAheadSectionContainer = styled.div`
  display: flex;
  margin: 10em 8em;
  padding-top: 10em;
  @media ${max.tabletLg} {
    margin: 10em 0;
  }
  @media ${max.tablet} {
    flex-wrap: wrap;
  }
`;`